website/docs/advanced/docker/deploy/manual.md

```diff
@@ -4,11 +4,317 @@ description: Build and Publish Images
+   necessary for testing purposes. In all these three cases, building a 
+   specific Docker image enables better testing and debugging in isolation.
+  </li>
+  <li><b>Build all the affected images</b>
+   in order to ensure the updated scripts and Dockerfiles are properly 
+   propagated through the nested hierarchy of GATK-SV Docker images.
+   Once you have successfully developed the specific Docker image and its 
+   encapsulated scripts, you will need to ensure that the changes are 
+   propagated throughout all the GATK-SV Docker images. 
+   This is necessary due to the nested hierarchy
+   of GATK-SV Docker images, where derived images should be rebuilt 
+   whenever any of their base images are updated. This process 
+   involves rebuilding all impacted images affected by the 
+   changes you made, ensuring the derived images are updated correctly, 
+   and integrating them into the pipeline.
+  </li>
+ </ul>
+</div>
+</details>
+:::
+
+
+### Image tag {#tag}
+
+[Docker image tags](https://docs.docker.com/engine/reference/commandline/tag/)
+are used to distinguish between different builds of the same image. 
+You can use any naming convention for your tags. 
+GATK-SV docker images use the following template for tags, 
+which you may want to adopt, in particular, if you plan to publish 
+your images on the GATK-SV container registries.
+
+```
+[Date]-[Release Tag]-[Head SHA 8]
+```
+
+where `[Date]` is `YYYY-MM-DD` extracted from the time stamp of the last
+commit on the feature branch, `[Release Tag]` is extracted from the latest [pre-]release on GitHub, 
+and the `[Head SHA 8]` is the first eight letters of the SHA of the 
+last commit on the feature branch.
+
+For example:
+
+```
+2023-07-28-v0.28.1-beta-e70dfbd7
+```
+
+For automatically composing image tags, you may follow the practices 
+used in [GATK-SV CI/CD](https://github.com/broadinstitute/gatk-sv/blob/286a87f3bcfc0b8c811ff789776dd0b135f582e9/.github/workflows/sv_pipeline_docker.yml#L85-L109).
+
+
+
+### Specify the container registry {#registry}
+The built images are stored on your computer. If you are only developing 
+or testing locally, there is no need to push them to a container registry. 
+In this case you can avoid providing `--docker-repo <registry>`.
+
+You need to push the images to a container registry if you want to:
+
+- Use the updated Docker images for WDL testing or development;
+- Store them on a container registry other than those maintained by the GATK-SV team.
+   
+The script automatically pushes Docker images to a container registry. 
+To use this feature, you may follow these steps:
+
+1. Ensure you are logged into Docker with credentials granting 
+push access to the container registry. Please refer to the 
+[Docker](#docker) section for details.
+
+
+2. Provide the `--docker-repo <registry>` argument, 
+replacing `<registry>` with the name of your container registry. 
+For Google Container Registry (GCR) and Azure Container Registry (ACR), 
```

Comment:
Good point! I added a note, but I think we should discuss and maybe update all the docs to refer to the artifact registry instead of GCR, in a separate PR maybe.